application/client/src/app/service/session/dependencies/search/store.ts

```diff
@@ -17,12 +17,12 @@ export type StoredEntity<T> = T & Hash & Recognizable & Updatable<EntityUpdateEv
 
 export abstract class Store<T> extends Subscriber {
     public subjects: Subjects<{
-        highlights: Subject<StoredEntity<T>[]>;
+        highlights: Subject<StoredEntity<T>>;
         value: Subject<StoredEntity<T>[]>;
         inner: Subject<StoredEntity<T>[]>;
         any: Subject<StoredEntity<T>[]>;
     }> = new Subjects({
-        highlights: new Subject<StoredEntity<T>[]>(),
+        highlights: new Subject<StoredEntity<T>>(),
```

Comment:
what is the motivation to change the signature of `highlights` event? Would it break other parts of solution?